pkgs/by-name/na/nautilus-open-in-blackbox/package.nix

```
@@ -0,0 +1,33 @@
+{ python3, fetchFromGitHub, gnome, stdenv, lib }:
+stdenv.mkDerivation rec {
+  pname = "nautilus-open-in-blackbox";
+  version = "0.1.1";
+
+  src = fetchFromGitHub
+    {
+      owner = "ppvan";
+      repo = pname;
+      rev = version;
+      hash = "sha256-5rvh3qNalpjamcBVQrnAW6GxhwPPlRxP5h045YDqvrM=";
+    };
+
+  patches = [ ./paths.patch ];
+
+  buildInputs = [
+    gnome.nautilus-python
+    python3.pkgs.pygobject3
+  ];
+
+  installPhase = ''
+    mkdir -p $out/share/nautilus-python/extensions
+    cp ./nautilus-open-in-blackbox.py $out/share/nautilus-python/extensions
```

```suggestion
    install -Dm555 ./nautilus-open-in-blackbox.py -t $out/share/nautilus-python/extensions
    
    runHook postInstall
```
and delete the `mkdir` line